File: Strivers_DSA_Playlist/1.Patterns_Beginners/Problem8/Problem8.java
import java.util.Scanner;

public class Problem8 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		        for(int i = 1; i <= n; i++) {
                int stars = 2*n - 2*(i-1) - 1;
                int spaces = i - 1;
                for(int j = 1; j <= spaces; j++) {
                        System.out.print(" ");
                }
                for(int j = 1; j <= stars; j++) {
                        System.out.print("*");
                }
                System.out.println();
        }
		scanner.close();
	}
}

File: Strivers_DSA_Playlist/1.Patterns_Beginners/Problem8/Problem8.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
	int n;
	cin >> n;

	for(int i = 1; i <= n; i++) {
		int stars = 2*n - 2*(i-1) - 1;
		int spaces = i - 1;
		for(int j = 1; j <= spaces; j++) {
			cout << " ";
		}
		for(int j = 1; j <= stars; j++) {
			cout << "*";
		}
		cout << endl;
	}

	return 0;
}
